docs/docs/visual-testing-with-storybook.md

```diff
@@ -170,13 +273,16 @@ First, create the story file. Storybook looks for all files with a `.stories.js`
 
 ```jsx:title=src/components/example.stories.js
 import React from "react"
-import { storiesOf } from "@storybook/react"
 
-storiesOf(`Dashboard/Header`, module).add(`default`, () => (
```

Comment:
I see the `storiesOf` API isn't deprecated yet, but will this code work in a backwards compatible fashion? https://storybook.js.org/docs/formats/storiesof-api/

Reply:
@marcysutton the `storiesOf` api will still be around for a bit as some other frameworks haven't still been ported to use CSF. More specifically the react native is still using it to declare the stories format and with that for Storybook to consume them. And CSF was introduced in version 5.2. Prior to that CSF approach will not work. Having both formats seems counter productive at this day and age. More even because the documentation is pointing to CSF usage.